.github/workflows/esrupgrade-common.yml

```diff
@@ -61,9 +61,9 @@ jobs:
             --env-file=dotenv/test.env \
             --env MM_SQLSETTINGS_DRIVERNAME="mysql" \
             --env MM_SQLSETTINGS_DATASOURCE="mmuser:mostest@tcp(mysql:3306)/mattermost_test?charset=utf8mb4,utf8&multiStatements=true&maxAllowedPacket=4194304" \
-            -v ~/work/mattermost:/mattermost \
-            -w /mattermost/mattermost \
-            $BUILD_IMAGE &
+            -v ~/work/mattermost:/mm \
+            -w /mm \
+            $BUILD_IMAGE&
```

Comment:
The space is gone here btw. Was that intentional?

Reply:
Whoops, nope, this is a side effect of me testing. Thanks!